e2e/helpers/html_helpers.py

```diff
@@ -0,0 +1,5 @@
+from bs4 import BeautifulSoup
+
+
+def make_soup(content):
+    return BeautifulSoup(content, 'html.parser')
```

Comment:
Don't think that there is a BeatifulSoup'ers community, I think it makes makes sense to inline it.